manubot/cite/tests/test_cite_command.py

```diff
@@ -65,38 +61,103 @@ def test_cite_command_file(tmpdir):
     not shutil.which('pandoc-citeproc'),
     reason='pandoc-citeproc installation not found on system'
 )
-def test_cite_command_render_stdout(args, expected):
+class Base_cite_command_render_stdout():
     """
-    Test the stdout output of `manubot cite --render` with various formats.
-    The output is sensitive to the version of Pandoc used, so rather than fail when
-    the system's pandoc is outdated, the test is skipped.
+    Expecting reference values for test to be at files on path:
+    cite-command-rendered/references-{format}-{pandoc_stamp}.{extension}
+
+    Examples:
+    - references-html-2.7.2.html  
+    - references-plain-2.7.2.txt
+    - references-jats-2.7.2.xml
+    - references-jats-2.7.3.xml
+    - references-markdown-2.7.2.md
+
+    2.7.2 is current pandoc version on CI builds for these builds.
+
+    Filenames must be adjusted accodingly when current pandoc version changes.
+
+    Run tests locally skipping this test suit (makes sense if your local pandoc
+    version is different from pandoc version used by Travis and Appveyor):
+
+        pytest -v -m "not pandocfail"
+        pytest -k "Test_cite_command_render_stdout" -m "not pandocfail"
```

Comment:
I'm not sure `pandocfail` is the best term, since this marker isn't applied to all tests that require pandoc. Maybe something like `pandoc-version-dependent`? But also we're not using this anywhere else. So for now would it just be easier for local users to use `pytest -k "not Test_cite_command_render_stdout"` and we'll include that in the docstring?

Reply:
I left a marker - it is safer for local user to depend on a marker, than requiring a user to know exact test name pattern to skip it. We already penalise a local tester with additional command line setups, so let's at least make that additional use sensible. 

Also need dearly a place to store developper manual,  #117. It will be rather hard to dig back to collect dev notes in retrospect.